cves/kernel/CVE-2014-7825.yml

```diff
@@ -450,7 +444,7 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: This was a lapse. The lapse was forgetting to check for invalid syscall numbers when tracing syscalls.
 CWE_instructions: |
   Please go to http://cwe.mitre.org and find the most specific, appropriate CWE
   entry that describes your vulnerability. We recommend going to
```

Comment:
Make sure to go back and confirm the CWE as well and change it to Manually confirmed.

Reply:
I missed this! Thanks, will do !